docs/examples/driver_examples/Qcodes example with Rigol DS1074 Z.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QCoDeS Example with the Rigol DS 1074 Z oscilloscope"
```

Comment:
can the model name be consistent in the notebook, python files, and names of the files? :) shall it just be DS1074Z everywhere?

Reply:
Wanted to say: "Thank you very much for your many lessons: Lord Mikhail" but the guy is immediately executed after this dialogue. So will keep it simple and in liking to your currently acquired Italian taste "Mille Grazie". Working on your comments!  @astafan8